### ISSUE TITLE
About the result in Traffic Junction

### ISSUE THREAD

**Issue by lucasliunju** [IC3Net/IC3Net#2]
Hi,

I'm very sorry to trouble you.  I am trying to run the code in Traffic Junction (easy version) and follow the arguments. But I find the result is not right. Finally, the success rate is about 23%. And I don' t know where the error is now. So I want to seek your help. Thanks!

**Comment by apsdehal** [comment]
Hi,
Can you post your system details and the command you are running here? it should be run on a machine with atleast 16 CPU cores to achieve similar. I can run your command on my end to verify.

**Comment by lucasliunju** [comment]
Hi,

Thanks for your reply.

I follow the command in README.md: python main.py --env_name traffic_junction --nagents 5 --nprocesses 16 --num_epochs 2000 --hid_size 128 --detach_gap 10 --lrate 0.001 --dim 6 --max_steps 20 --ic3net --vision 0 --recurrent --add_rate_min 0.1 --add_rate_max 0.3 --curr_start 250 --curr_end 1250 --difficulty easy

As for the system, CentOS 7, CPU(s): 32，Core(s) per socket: 8， Intel(R) Xeon(R) CPU E5-2620 v4 @ 2.10GHz

Thanks!

**Comment by apsdehal** [comment]
Hey @lucasliunju ,
Sorry for the delay in response. I just ran the exact query (copy-pasted from readme) and seems to work for me.
I would probably need more time to understand why it isn't working for you. What PyTorch version are you using?
![image](https://user-images.githubusercontent.com/3616806/56171103-16a15500-5fb2-11e9-9e50-41d4efaf6113.png)

**Comment by lucasliunju** [comment]
Hi, the result is similar with me. However, when the add_rate increases, the model can not converge.

**Comment by apsdehal** [comment]
Hi,
I found the issue, and I have updated the code to match with the settings described in the paper. As described there, we set comm action to 1 in case of TJ to test effect of individual rewards on cooperative scenarios in an isolated scenario by lowering the burden to learn the comm action. (I think we made this mistake while open sourcing the code). Thanks for pointing this out, I really appreciate it.
